tfjs-backend-webgl/src/backend_webgl.ts

```diff
@@ -688,27 +688,30 @@ export class MathBackendWebGL extends KernelBackend {
     return true;
   }
 
-  private releaseGPUData(dataId: DataId): void {
+  private releaseGPUData(dataId: DataId, isRead = false): void {
     const {texture, dtype, texShape, usage, isPacked, slice} =
         this.texData.get(dataId);
     const key = slice && slice.origDataId || dataId;
     const refCount = this.dataRefCount.get(key);
 
     if (refCount > 1) {
+      // When reading a tensor which has been shallow sliced, do nothing.
+      if (slice == null && isRead) {
```

Comment:
It seems that we shouldn't call [releaseGPUData](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/backend_webgl.ts#L1195) for read. Otherwise, for `shallowSlice` tensor, we always meet problems like below case: (`tensor.data()` and `tensor.dispose()` will decrease tensor's ref count twice for the same tensor. If this tensor has a shallow sliced tensor, we will meet problem when access the shallow tensor's content since the underlying texture has been released.).

@lina128 @ping, can we just simply remove [releaseGPUData](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgl/src/backend_webgl.ts#L1195) there to fix this issue?